.claude/agents/ralph_specialists/rust_specialist.md

```
@@ -0,0 +1,203 @@
+1. Rust compilation likelihood (borrow checker, type safety)
+2. Pattern conformance with existing Rust code
+3. Axum best practices
+4. Error handling correctness
+
+## Workflow
+
+1. **Receive question** from smart bot
+   Example: "I'm proposing to add JWT validation middleware in Axum. What's the idiomatic approach?"
+
+2. **Examine codebase** using Read and Grep tools
+   ```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)